src/Semantic/Git.hs

```diff
@@ -32,24 +41,29 @@ clone url path = sh $ do
 -- | git cat-file -p
 catFile :: FilePath -> OID -> IO Text
 catFile gitDir (OID oid) = sh $ do
-  run "git" ["-C", pack gitDir, "cat-file", "-p", oid]
+  run "git" ["-C", pack gitDir, "cat-file", "-p", Text.decodeUtf8 oid]
 
 -- | git ls-tree -rz
 lsTree :: FilePath -> OID -> IO [TreeEntry]
-lsTree gitDir (OID sha) = sh $ parseEntries <$> run "git" ["-C", pack gitDir, "ls-tree", "-rz", sha]
+lsTree gitDir (OID sha) = Streaming.Process.withStreamingOutput lsproc go
+  where
+    lsproc = Streaming.Process.proc "git" ["-C", gitDir, "ls-tree", "-zr", BC.unpack sha]
```

Comment:
Is there any semantic difference implied by the change from `-rz` to `-zr`?

Reply:
I don’t think so, but I’ll change it back.